test/fixtures/code_samples/App.vue

```diff
@@ -0,0 +1,44 @@
+<style scoped>
+  label {
+    display: block;
+    margin: 1em 0;
+    width: 20em;
+    font-size: 0.8em;
+  }
+  input {
+    width: 10em
+  }
+  div {
+    width: 7em;
+    text-align: center;
+  }
+</style>
+
+<script setup>
+import { ref } from 'vue'
+
+const kms = ref(1.609344)
+const miles = ref(1)
+
+function setKms(e, v = +e.target.value) {
+  kms.value = v
+  miles.value = v / 1.609344
+}
+
+function setMiles(e, v = +e.target.value) {
+  miles.value = v
+  kms.value = v * 1.609344
+}
```

Comment:
Could you add a single line comment and also a block comment anywhere here for tests purpose ?

Reply:
Sure thing, done!